frontend/src/app/pages/about-me/about-me.component.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  margin: 0;
+}
+
+.items {
+  padding: 2rem;
```

Comment:
Do not user hardcoded value. import from scss variables then use the appropriate variable. e.g. $padding-small